website/docs/d/is_virtual_endpoint_gateway.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_virtual_endpoint_gateway"
+sidebar_current: "docs-ibm-datasource-is-virtual-endpoint-gateway"
+description: |-
+  Manages IBM Cloud Infrastructure virtual endpoint gateway .
+---
+
+# ibm_is_virtual_endpoint_gateway
+
+Import the details of an existing IBM Cloud Infrastructure virtual endpoint gateway  as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_is_virtual_endpoint_gateway" "data_test" {    
+    name = ibm_is_virtual_endpoint_gateway.endpoint_gateway.name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- name(Required,string)- Endpoint gateway name
```

Comment:
format is wrong. check the other attribute and change it.

Reply:
handled